components/automate-ui/src/app/pages/team-details/team-details.component.ts

```diff
@@ -124,7 +133,7 @@ export class TeamDetailsComponent implements OnInit, OnDestroy {
       takeUntil(this.isDestroyed)
     ).subscribe(([gStatus, uStatus, usersStatus]) => {
       this.isLoadingTeam =
-        (gStatus !== EntityStatus.loadingSuccess) ||
+        (gStatus === EntityStatus.loading) ||
         (uStatus === EntityStatus.loading) ||
         (usersStatus === EntityStatus.loading);
```

Comment:
Suggest use the new `allLoadedSuccessfully` function we introduced in Lance's PR (see https://github.com/chef/automate/pull/2151).
Also, would be more consistent with most other files to change `isLoadingTeam` to `loading`.

Reply:
trying to scope down the PR so i'll make a note of that for a quick follow-up PR